packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 19.0.0
+* **Breaking Change** [swift] Do not use `FlutterError` on the generated code.
```

Comment:
Should be an empty line between these.

`* **Breaking Change** [swift] Removes `FlutterError` in favor of `PigeonError`.`

Reply:
Done.

Reply:
can you update the changelog itself to match my comment?